build/MPack.targets

```diff
@@ -1,12 +1,37 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageDependsOn Condition="'$(OS)'=='Windows_NT'">$(PackageDependsOn);GenerateMPack</PackageDependsOn>
     <AddinName>Microsoft.VisualStudio.Mac.RazorAddin</AddinName>
     <AddinDirectory>$(RepositoryRoot)tooling\$(AddinName)\</AddinDirectory>
+    <MPackArtifactCategory>shipoob</MPackArtifactCategory>
```

Comment:
noship?

Reply:
Eh, gonna nuke the file entirely.

Reply:
Why not just nuke all of MPack code then?

Reply:
> Why not just nuke all of MPack code then?

That's what I meant 😄

Reply:
seems like a potentially painful merging strategy, but if it is what your team wants, :shipit: